app/controllers/topics_controller.rb

```diff
@@ -2,4 +2,23 @@ class TopicsController < ApplicationController
   def index
     @topics = Topic.all
   end
+
+  def new
+    @topic = Topic.new
+  end
+
+  def create
+    @topic = Topic.create(topic_params)
+    if @topic.valid?
```

Comment:
This isn't right. It will should be throwing an error in your test.
`Topic.create` will return either the topic if the creation succeeds, or will return nothing (i.e. `nil`) if the creation fails, and `.valid?` is not a method you can call on `nil`.

Here's my console output to show you what's happening when the topic is invalid:

<img width="707" alt="screen shot 2018-02-26 at 12 46 28" src="https://user-images.githubusercontent.com/977566/36671369-46e832f6-1af3-11e8-98e7-fc7e17f16cde.png">